assemblies/assembly-install-rhdh-ocp.adoc

```diff
@@ -1,19 +1,31 @@
 [id='assembly-install-rhdh-ocp']
 
-= Deploying {product} on {ocp-short}
+= Installing {product} on {ocp-short}
+:context: install-rhdh-ocp
 
 You can install {product} on {ocp-short} using one of the following methods:
 
-* The Helm chart
-* The {product} Operator
+* *The {product} Operator*
+** Ready for immediate use in {ocp-short} after it is installed by an administrator
+** Available for use in Kubernetes
+** Updates automatically
+** Managed by Operator Lifecycle Management (OLM)
 
-// Helm chart method
-include::modules/installation/proc-install-rhdh-ocp-helm.adoc[leveloffset=+1]
-include::modules/getting-started/proc-add-custom-app-file-openshift-helm.adoc[leveloffset=+2]
-include::modules/installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc[leveloffset=+2]
+* *The {product} Helm chart*
+** Works seamlessly in both {ocp-short} and Kubernetes
+** Requires manual installation and management
```

Comment:
@linfraze would it make sense to expand this bullet to better contrast (if I'm reading this right) with the "Updates automatically" and "Managed by OLM" bullets mentioned in the operator installation method?  And maybe also with the first bullet, I wonder if it would be more clear to expand these two points out into separate bullets.

Reply:
@gashcrumb thanks for the feedback. It might be better to collapse the auto-update and OLM bullets into a single bullet rather than breaking the other one into two. Are you also suggesting that we break the bullet about working on both OCP and K8s into two? I'd like to better understand why that would give more clarity without adding unnecessary fluff. There might also be additional details that we can include that would further justify breaking the bullet into two.

Reply:
It seemed like these bullets sort of compare/contrast against the bullets listed under the operator based install method, and so I here think I'm equating clarity with an equal number of bullets for each method.  Which may not actually be the right direction, so figured I'd raise it and see what y'all thought :smile:

Reply:
Applied your feedback, thanks!